plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/connection/IOpenShiftConnection.java

```diff
@@ -38,6 +38,8 @@
 	
 	<T extends IResource> List<T> getResources(String kind, String namespace);
 
+	<T extends IResource> T getResource(IResource resource);
```

Comment:
I dont remember this methods purpose?  Is it to refresh a resource?  or to load a resource from basically a predicate?  should we call it something else?